thirdparty/faiss/faiss/IndexIVFPQ.cpp

```diff
@@ -872,59 +872,73 @@ struct IVFPQScannerT : QueryTables {
 
     /// version of the scan where we use precomputed tables
     template <class SearchResultType>
-    void scan_list_with_table(
```

Comment:
Same here, can we don't change the interface?

Reply:
good catch, has changed it back